### ISSUE TITLE
How do we work out what we can throw away?  Or better storage policy?

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#296]
As we're getting busier, space is becoming more and more of an issue in the workshop.

There are lots of random cardboard boxes under the tables, so now we're accumulating random cardboard boxes on top of the tables too :-D

Some have address labels on, lots don't.  It's not clear if they're for general use, things people are expecting to store safely, things to donate to the general workshop supplies, or just rubbish...
- There's a cardboard box full of other boxes, packaging materials, and string on the tables in the middle
- A cardboard box containing a computer case under the tables
- A cardboard box of glue sticks under the tables (these are now in with the other glue gun supplies)
- A cardboard box with a clear plastic box (trainer box, it seems)
- A cardboard box addressed to @DoESsean with two big rectangular blocks of some beige material inside (from Easy Composites)
- A cardboard box addressed to @DefProc full of rolls of vinyl
- A Playstation box (with manual and polystyrene packaging, but no Playstation)

Mostly it's a failure of communication - I think we're getting to a size where we could do with a mechanism for sorting it out, or a policy for how $random_person_helping_tidy_up can decide what should/shouldn't be kept.

(And that's before we start on the piles of wood generally getting in the way of the drawers everywhere... :-)

**Comment by amcewen** [comment]
The standard pattern used in other hackspaces is to have "do not hack" labels that have to be attached to anything you want to keep (and have your name/contact details and the date you'll remove them by).  Anything else is then fair game for someone to (a) use themselves or (b) throw away.

London Hackspace also has a "Notice of disposal" system, where larger items with no "do not hack" label on them get a warning before they're disposed of.  They also have a "storage request" system that people have to use before they can leave anything bigger than stuff which fits in their member's box in the space.

Nottinghack have (I think) an oversize storage area, where projects that don't fit into your member's box can be left - I'm not sure of the rules around how to bring stuff in or how long you can keep it.

I don't know if those are the best patterns for us to adopt, but I think we need to work out something.  Paying for extra storage is something we can look at too - @johnmckerrell already pays for more storage, and a few others have asked about that in the past... (maybe more for when we move and have more space that we can rent out :-D)

**Comment by amcewen** [comment]
Nottinghack's storage rules are detailed at http://rules.nottinghack.org.uk/en/latest/storage-in-the-hackspace.html

**Comment by amcewen** [comment]
This is a duplicate of #538 (well, strictly speaking #538 is a duplicate of this, but that's a better expressed list of what we should be doing)
